src/storage/v2/transaction.hpp

```diff
@@ -113,7 +113,21 @@ struct Transaction {
   rocksdb::Transaction *disk_transaction_;
   /// Main storage
   utils::SkipList<Vertex> vertices_;
-  std::vector<std::unique_ptr<utils::SkipList<Vertex>>> index_storage_;
+
+  using LabelIndex = LabelId;
+  using LabelPropertyIndex = std::pair<LabelId, PropertyId>;
+  using LabelPropertyValIndex = std::tuple<LabelId, PropertyId, PropertyValue>;
```

Comment:
Not sure if caching by property value makes sense -> connected to one of the comments above